.github/renovate.json5

```diff
@@ -112,5 +112,18 @@
       ],
       datasourceTemplate: 'npm',
     },
+    {
+      customType: 'regex',
+      fileMatch: ['.md$'],
```

Comment:
Just thinking out loud: would we want to ignore `CHANGELOG.md` entirely, particularly if we apply this to a Buildkite plugin repo in future?